explainaboard_client/config.py

```diff
@@ -15,15 +15,15 @@ class HostConfig:
     HostConfig,
     {
         "main": HostConfig(
-            host="https://explainaboard.inspiredco.ai/api",
+            host="https://explainaboard.inspiredco.ai",
             frontend="https://explainaboard.inspiredco.ai",
         ),
         "staging": HostConfig(
-            host="https://dev.explainaboard.inspiredco.ai/api",
+            host="https://dev.explainaboard.inspiredco.ai",
             frontend="https://dev.explainaboard.inspiredco.ai",
         ),
         "local": HostConfig(
-            host="http://localhost:5000/api", frontend="http://localhost:3000"
+            host="http://localhost:5000", frontend="http://localhost:3000"
```

Comment:
Sorry this is out of the topic in this PR, but these configs shouldn't be exposed externally. We shouldn't expose any environmental information used inside the company to hide the details of our workflow.

Reply:
I think we need to clean up this repository.